packages/cli/src/api/catalog.ts

```diff
@@ -227,26 +227,48 @@ export class Catalog {
     catalogs: Object,
     locale: string,
     key: string,
-    { fallbackLocale, sourceLocale }: GetTranslationsOptions
+    { fallbackLocales, sourceLocale }: GetTranslationsOptions
   ) {
     if (!catalogs[locale].hasOwnProperty(key)) {
       console.error(`Message with key ${key} is missing in locale ${locale}`)
     }
 
     const getTranslation = (locale) => catalogs[locale][key].translation
 
+    const getMultipleFallbacks = (locale) => {
+      let i = 0
+      const fL = fallbackLocales[locale]
+
+      // some probably the fallback will be undefined, so just search by locale
+      if (!fL) return null
+
+      if (Array.isArray(fL)) {
+        while(i <= fL.length) {
+          const fallbackLocale = fL[i]
+          if (catalogs[fallbackLocale]) {
+            return getTranslation(fallbackLocale)
+          }
+          i++
+        }
```

Comment:
```suggestion
        for(const fallbackLocale of fL) {
          if (catalogs[fallbackLocale]) {
            return getTranslation(fallbackLocale)
          }
        }
```

Could we remove the `while` loop here?